SpaceShooterGame/SpaceShooterGame/GameScene.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GameScene.swift
+//  SpaceShooterGame
+//
+//  Created by Tristan Garzon on 2018-03-07.
+//  Copyright © 2018 Tristan Garzon. All rights reserved.
+//
+
+
+//Notes: Timer, Enemies, Firing
+
+import SpriteKit
+import GameplayKit
+
+class GameScene: SKScene, SKPhysicsContactDelegate {
+    
+    
+    var gbackground:SKEmitterNode!
+    var player:SKSpriteNode!
+    
+    //Score
+    var scoreLabel:SKLabelNode!
+    var score:Int = 0 {
+        didSet {
+            scoreLabel.text = "Score: \(score)"
+        }
+    }
+    
+    override func didMove(to view: SKView) {
+        
+        //Background (File, Position, Time)
+        gbackground = SKEmitterNode(fileNamed: "Background")
+        gbackground.position = CGPoint(x: 0, y: 1472)
+        gbackground.advanceSimulationTime(10)
+        self.addChild(gbackground)
+        
+        gbackground.zPosition = -1
+        
+        //Player (File, Scale, Position, Add)
+        player = SKSpriteNode (imageNamed: "redfighter0006")
+        player.setScale(0.4)
```

Comment:
Should the game scene know all these implementation details about the player or should that be handled by another class?